fuel-vm/src/checked_transaction.rs

```diff
@@ -945,7 +964,6 @@ mod tests {
     // input
     #[quickcheck]
     fn min_fee_message_input(
```

Comment:
The purpose of this test to test different `gas_price` as well, could you return it back, please?=)